src/parser.js

```diff
@@ -3,31 +3,24 @@
 const spawnSync = require("child_process").spawnSync;
 const path = require("path");
 
-function parseText(text, pythonExecutable) {
-  const executionResult = spawnSync(
-    pythonExecutable,
-    [path.join(__dirname, "../vendor/python/astexport.py")],
-    {
-      input: text
-    }
-  );
-
-  const error = executionResult.stderr.toString();
-
-  if (error) {
-    throw new Error(error);
+function parse(text) {
+  const executionResult = spawnSync("python", ["-m", "prettier.parser"], {
```

Comment:
you get a bit more flexibility when you run something as a module... you can do things like relative imports, which are sorta handy